alchemist-loading/src/test/resources/testCSVExporter.yml

```diff
@@ -0,0 +1,44 @@
+incarnation: protelis
+variables:
+  zoom: &zoom
+    formula: 0.1
+  image_name: { formula: "'chiaravalle.png'" }
+  image_path: &image_path
+    language: kotlin
+    formula: >
+      import java.io.File
+      File("../..").walkTopDown().find { image_name in it.name }?.absolutePath ?: image_name
+  walking_speed: &walk-speed { default: 1.4, min: 1, max: 2, step: 0.1 }
+  seed: &seed { default: 0, min: 0, max: 99, step: 1 }
+  scenario_seed: &scenario_seed { formula: (seed + 31) * seed }
+  people_count: &people_count
+    type: GeometricVariable
+    parameters: [300, 50, 500, 9]
+seeds: { simulation: *seed, scenario: *scenario_seed}
+export:
+  - type: CSVExporter
+    parameters: ["00-testing_csv_export"]
```

Comment:
uhmm... does this output into the simulator folder? If so, it may create a mess: we can instead create a temporary directory, I can help you do so.

Reply:
Done.